src/index.html

```diff
@@ -17,16 +17,28 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar primary-search-bar"
+      data-qa="big"
+    >
+      <input
+        class="search-bar-input search-bar-input--primary"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
```

Comment:
Good job using the `placeholder` attribute for user guidance. However, consider making the placeholder text more generic and not location-specific. Something like 'Search...' could work better.